docs/DeveloperGuide.md

```diff
@@ -312,16 +312,79 @@ Priorities: High (must have) - `* * *`, Medium (nice to have) - `* *`, Low (unli
 
 ### Use cases
 
-(For all use cases below, the **System** is the `AddressBook` and the **Actor** is the `user`, unless specified otherwise)
+(For all use cases below, the **System** is the `TutorTrack`, the **Actor** is the `Tutor` and the **Person** is the `Tutee`, unless specified otherwise)
+
+**Use case: View all Person**
+
+**MSS**
+
+1.  User requests to list all persons
+2.  System shows a list of all persons
+
+    Use case ends.
+
+**Extensions**
+* 2a. The list is empty.
+* *a. Connection lost.
```

Comment:
Not super sure, but I think we might not need an extension for connection lost at every use case. I think it might be a bit tedious to write at every use case.